src/containers/NoAccessPage/NoAccessPage.js

```diff
@@ -18,8 +18,13 @@ import {
   Page,
   ResponsiveBackgroundImageContainer,
   LayoutSingleColumn,
+  NamedLink,
+  ExternalLink,
 } from '../../components';
 
+import { generateLinkProps } from '../../util/routes';
```

Comment:
Move this after '../../util/urlHelpers'; 
(util/routes imports util/urlHelpers)